tardis/io/tests/test_HDFWriter.py

```diff
@@ -79,7 +77,8 @@ def test_multi_index_write(tmpdir):
     actual.to_hdf(fname, path="test", overwrite=True)
     expected = pd.read_hdf(fname, key="/test/mock_hdf/property")
     expected = pd.MultiIndex.from_tuples(expected.unstack().values)
-    pdt.assert_almost_equal(actual.property, expected)
+    # These are multiindex objects, so we need to compare the values
+    np.all(expected.values == actual.property.values)
```

Comment:
Does this actually test anything? I think this test is missing an assert.

Reply:
correct.